src/main/java/com/hackathonorganizer/hackathonwriteservice/hackathon/controller/HackathonController.java

```diff
@@ -0,0 +1,43 @@
+package com.hackathonorganizer.hackathonwriteservice.hackathon.controller;
+
+import com.hackathonorganizer.hackathonwriteservice.hackathon.model.Hackathon;
+import com.hackathonorganizer.hackathonwriteservice.hackathon.model.HackathonDto;
+import com.hackathonorganizer.hackathonwriteservice.hackathon.service.HackathonService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/hackathon")
+@AllArgsConstructor
+public class HackathonController {
+
+    private final HackathonService hackathonService;
+
+    @PostMapping(consumes = "application/json")
+    public ResponseEntity<HackathonDto> createHackathon(@RequestBody Hackathon hackathon) {
```

Comment:
Czemu response entity? Byłbym za tym żeby tego nie używać tylko adnotacji

Reply:
Chodzi o @ResponseBody jak w takim wypadku ustawic status code?

Reply:
https://www.baeldung.com/spring-response-status
Jeśli byłby inny status typu bad request to wyłapiemy go w serwisie i, wtedy zwrócimy bad request, a domyślnie będzie zwracać created